abstract_specification.md

```diff
@@ -1,31 +1,27 @@
 # Introduction
 
-> **Status**: in active development. Version: `0.2.0-dev`
->
-> ℹ️ **Note** - this is an early working draft (WORK IN PROGRESS). It may contain controversial, undecided, or incomplete proposals - feedback is encouraged. It is expect that additional revisions will be required to converge the underlying primitives with use cases, and to incorporate lessons from initial implementation and early users. Please see `issues` and other call-outs throughout the doc for a list of active discussions/debates.
-
 The goal of SOMA (“stack of matrices, annotated”) is a flexible, extensible, and open-source API providing access to annotated, 2D matrix data stored in multiple underlying formats and systems. The vision for this API family includes:
 
-- support access to persistent, cloud-resident datasets,
-- enable use within popular data science environments (e.g., R, Python), using the tools of that environment (e.g., Python Pandas integration),
-- enable "out-of-core" access to data aggregations much larger than single-host main memory
-- enable distributed computation over datasets
-- provide a building block for higher-level API that may embody domain-specific conventions or schema around annotated 2D matrices (e.g., a cell "atlas").
+* support access to persistent, cloud-resident datasets,
+* enable use within popular data-science environments (e.g. R, Python), using the tools of that environment (e.g. Python Pandas integration),
+* enable "out-of-core" access to data aggregations much larger than single-host main memory,
+* enable distributed computation over datasets, and
```

Comment:
Looking at this and the changes below I'm inclined to think that, absent a compelling reason that the bullets build on each other specifically, we should just use each one as its own standalone unit rather than trying to string them together:

```markdown
Our primary goals are to:

- eat lunch
- build a dangerous contraption
- yell at passersby and make them uncomfortable
```